config/application.rb

```diff
@@ -1,4 +1,5 @@
 require_relative 'boot'
+require_relative '../app/lib/core_ext/object'
```

Comment:
I think this does belong in `/lib`, and `/lib` in is the load path but not the auto load path - so can be loaded from `/lib` with `require 'core_ext/object'`